### ISSUE TITLE
Games with similar names not detected

### ISSUE THREAD

**Issue by sdrubble** [burnhamup/galaxy-integration-indiegala#8]
I have the following THREE games on IG:

- Waste Walkers
- Waste Walkers Deliverance
- Waste Walkers Subsistence

Only the first one is shown in Galaxy - the 2nd and 3rd ones are NOT.

IIRC, someone else complained somewhere in GOG forums about this same exact issue awhile ago.

**Comment by burnhamup** [comment]
This may be an issue with GOG's game database. The only thing I can tell Galaxy is the name of the game. On Steam, Waste Walkers Deliverance and Waste Walkers Subsistence are considered DLC and not as unique games, so I guess the same is true of the Galaxy DB.

**Comment by EustaceCS** [comment]
I believe Steam is not the main source of intel regarding which titles _are_ games (and known) and which _are not_ for GOG database.
Bumped into it with itch.io integration, with few titles which _were_ allowed to reach Steam but were _not_ recognized by integration/Galaxy/Galaxy's base.
Using Fallout 76 as a benchmark, trying to Add it Manually reveals that something is not right in Galaxy's database (because since when Deluxe Editions became stand-alone titles? dat pepesda...)
![Capture](https://user-images.githubusercontent.com/29461793/82353271-dc518500-9a07-11ea-86d6-ccbecdf74fa8.PNG)
In other words, if you believe that it's a separate title - stick it as a separate title and poke GOG support if it won't bulge :) It's not integration's issue, it's Galaxy games database's shenanigans.

**Comment by jonisb** [comment]
As far as I know GOG Galaxy 2.0 uses https://www.igdb.com for game metadata.

**Comment by burnhamup** [comment]
Closing this since it has to do with Galaxy proper and the https://www.igdb.com database which I can't control with the plugin
